nextpy/frontend/components/component.py

```diff
@@ -1590,6 +1590,8 @@ def _get_memoized_event_triggers(
             if isinstance(rendered_chain, str):
                 rendered_chain = rendered_chain.strip("{}")
 
+            rendered_chain = rendered_chain+'}'
```

Comment:
Why is this change made? Isn't `rendered_chain` working correctly previously?

Reply:
I've noticed an issue with the original ```rendered_chain``` definition in the component.py file. The initial code is as follows:
```
{(_e0) =>{ 
    const dispatcher = dispatchers['state.state'];
    dispatcher({ message: _e0.target.value });
    addEvents([Event("state.state.set_message", {value:_e0.target.value})], _e0, {}) 
}}
```
However, there seems to be a processing step that alters rendered_chain by removing the opening and closing curly brackets ```{}```. This modification necessitates manually appending an additional closing curly bracket ```}``` at the end of the ```rendered_chain``` to ensure correct rendering in the index.js file. The modified code looks like this:
```
_e0 => {
    const dispatcher = dispatchers['state.state']
    dispatcher({ message: _e0.target.value })
    addEvents([Event('state.state.set_message', { value: _e0.target.value })], _e0, {})
},
```
This workaround fixes the rendering issue.

Reply:
The ```rendered_chain``` is working correctly and when you print you get
``` (_e0) =>{ const dispatcher = dispatchers['state.state'];
dispatcher({ message: _e0.target.value });
addEvents([Event("state.state.set_message", {value:_e0.target.value})], (_e0), {}) ```
but there the issue is when it is rendered in the index.js file. It is being rendered inside a function like below 
```
const on_change_b21e2d85fd474865cae05766137a2912 = useCallback(
		_e0 => {
			const dispatcher = dispatchers['state.state']
			dispatcher({ message: _e0.target.value })
			addEvents([Event('state.state.set_message', { value: _e0.target.value })], _e0, {})
		},
		[addEvents, Event]
``` and here, this function previously was being written in single line and so it doesn't had any closing bracket at the end but now we need a closing bracket at the end.